_rest_routes.inc.php

```diff
@@ -417,10 +416,20 @@
         return (new FhirPractitionerRestController())->patch($id, $data);
     },
     "GET /fhir/Organization" => function () {
-        return (new FhirOrganizationRestController(null))->getAll($_GET);
+        return (new FhirOrganizationRestController())->getAll($_GET);
     },
     "GET /fhir/Organization/:id" => function ($id) {
-        return (new FhirOrganizationRestController(null))->getOne($id);
+        return (new FhirOrganizationRestController())->getOne($id);
+    },
+    "POST /fhir/Organization" => function () {
+        RestConfig::authorization_check("admin", "users");
+        $data = (array) (json_decode(file_get_contents("php://input"), true));
+        return (new FhirOrganizationRestController())->post($data);
+    },
+    "PATCH /fhir/Organization/:id" => function ($id) {
```

Comment:
just noted this. for the above 2 endpoints should have:
```
RestConfig::authorization_check("admin", "super");
```
(then will be consistent with the api permissions)